cmd/surf/commands/redact.go

```diff
@@ -79,6 +81,11 @@ func (cmd *RedactWithExtractorCmd) initWithArgs(args *redactWithExtractorArgs, f
 		return err
 	}
 
+	if !config.IsOutputRedirected() &&
+		!flags.IsOutputSpecified() {
+		return errors.New("you must use '-o' or '-m' or redirect stdout when redacting files")
```

Comment:
Could we do `-o` automatically? E.g. if no filename supplied then write to `redacted.pdf` or something similar.